core/metacling/src/TCling.cxx

```diff
@@ -1964,7 +1976,15 @@ void TCling::RegisterModule(const char* modulename,
       }
    }
 
-   if (gIgnoredPCMNames.find(modulename) == gIgnoredPCMNames.end()) {
+   // This is an executable. There is no point in trying to load rdict and pcm
```

Comment:
The "virtual readers' instruction pointer" can arrive at the comment even if the library is a shared library and not an executable. Maybe reword the comment: "If the library was not dlopened it's likely an executable. There is no point..."